packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -0,0 +1,364 @@
+<template>
+    <div
+        ref="cookieBanner"
+        :class="[$style['c-cookieBanner'], $style['c-cookieBanner-overlay'], { [$style['c-cookieBanner--hidden']]: hideBanner }]"
+        data-cookie-consent-overlay
+        data-test-id="cookieBanner-component"
+        :aria-hidden="hideBanner">
+        <div
+            v-if="!legacyBanner"
+            :class="[$style['c-cookieBanner-card'], { [$style['c-cookieBanner-ios']]: checkIosSafari() }]">
+            <div
+                :class="$style['c-cookieBanner-content']"
+                role="dialog"
+                aria-labelledby="cookieConsentLabel"
+                aria-modal="true">
+                <h2
+                    id="cookieConsentLabel"
+                    :class="$style['c-cookieBanner-title']">
+                    <a
+                        ref="cookieBannerHeadingFocus"
```

Comment:
The name is a bit confusing, in my opinion. The `ref` is to reference the element, and "Focus" seems more to do with what you're going to do with the `ref` in a method than what the element is about. Dropping the "focus" on this `ref` makes more sense for me.

Reply:
Done